content/integrations/awsdirectconnect.md

```diff
@@ -8,9 +8,9 @@ doclevel: basic
 
 ## Overview
 
-AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+This integration collects metrics from AWS Direct Connect (e.g. `aws.dx.connection_state`, `aws.dx.connection_bps_egress`)
```

Comment:
I am looking for human-readable examples here. We've got the metrics section below for fully qualified metric names, e.g. "This integrations collect metrics from AWS Direct Connect like connection states, BPS egress, and more."

Reply:
Great call, should have realized that.